### ISSUE TITLE
Unyson 2.7.5 (WP 4.8.1) -- Content Missing from Page Editor

### ISSUE THREAD

**Issue by marine207** [ThemeFuse/Unyson#2773]
Recently updated Unyson, now I cannot edit pages.

What I've noticed so far:
-When trying to edit a page, no "text fields" present to view/edit the page's content
-When trying to edit a page, no "Update" button (only the "Preview Changes" button)
-When clicking "Settings" for Page Builder (from the Unyson tab), no content visible except "Save" and "Cancel"

If I disable Unyson plugin, I'm able to view and edit the page's content, but obviously I get some formatting oddities on my site.  Reactivating the plugin puts my site back to normal, but puts me right back to not being able to edit pages.

Anyone else with this issue?

**Comment by marine207** [comment]
Seems like many having this issue.  Hope a solution is implemented soon.

**Comment by marine207** [comment]
2.7.6 was just released. Closing the issue now.  More information here: https://github.com/ThemeFuse/Unyson/issues/2764

**Comment by danyj** [comment]
@marine207 is being monitored did you update to 2.7.6 ?

**Comment by mtmcgreg** [comment]
When I go to a page, the information from before is there and displayed. However when I go to edit it, the information is not there so it isn't editable. As you can imagine this is a big problem. I need to be able to directly edit each page and the existing content. 

I have updated to 2.7.6

**Comment by marine207** [comment]
@danyj I have updated to 2.7.6 just a few seconds ago.  Doesn't appear to have been fixed entirely.  Restarting browser and clearing cache to try that.

**Comment by marine207** [comment]
<img width="1109" alt="unyson_issue" src="https://user-images.githubusercontent.com/31111312/29432002-37fffdcc-83af-11e7-86ac-ce7a66c103fe.png">

**Comment by danyj** [comment]
@marine207 @mtmcgreg 

see this please 
https://github.com/ThemeFuse/Unyson/issues/2764#issuecomment-323184014

provide info to @GheorgheP  so that we can get to the bottom of it

**Comment by mtmcgreg** [comment]
I sent

**Comment by danyj** [comment]
@marine207 by looking at your image im sure there are JS errors in console , that causes the blank  boxes

**Comment by marine207** [comment]
@danyj how do I check for JS errors?  What credentials does gheorgheP need?

**Comment by danyj** [comment]
@marine207  email @GheorgheP  send him site link and admin login

console 
https://developers.google.com/web/tools/chrome-devtools/console/

**Comment by marine207** [comment]
@danyj there are definitely JS errors.  many of them.  I have sent credentials to him

**Comment by danyj** [comment]
see this 
https://github.com/ThemeFuse/Unyson/issues/2764#issuecomment-323189456

**Comment by marine207** [comment]
@danyj updated and no longer having issues.  Thank you!!

**Comment by ia-code** [comment]
I found the _arrangement_ of the various wp dialogues matters.  I have _two_ wysiwyg editors on my page, and when I put wp dialogues **between** them the UNYSON extension vanished.  I moved all the wp dialogues _below_ my editors and it returened on page reload.  I did not try all the wp dialogs above both editors.  The odd thing is another post type has other wp dialogues (not editors) between two editors and works fine.  

The commonality is that in both I have WooCommerce Membership _below_ other dialogues.  I have WooCommerce Sensei below on one post type and between on the other.  My Advanced Custom Fields are above the editors.

IMHO, this is programatic nonsense!
